test/powershell/engine/HelpSystem.Tests.ps1

```diff
@@ -65,3 +74,65 @@ Describe "Validate Get-Help for all cmdlets in 'Microsoft.PowerShell.Core'" -Tag
 
     RunTestCase -tag "Feature"
 }
+
+# 1. WSMan:\ and Cert:\ providers are not yet supported on non-Windows platforms.
+# 2. Update-Help is not yet supported on non-Windows platforms; currently it is using
+#    Windows Cabinet API (cabinet.dll) internally.
+if ($IsWindows)
+{
```

Comment:
Instead of skipping here. Please skip in "It" using -skip (-not $IsWindows) {... }. This makes sure that we log that these tests are skipped.